src/site/content/en/blog/sms-otp-form/index.md

```diff
@@ -0,0 +1,266 @@
+primary identifier. In such services, users can enter their phone number and the
+OTP received via SMS to prove their identity. Sometimes it's combined with a PIN
+to constitute a two-factor authentication.
+* **Account recovery.** When a user loses access to their account, there needs
+to be a way to recover it. Sending an email to their registered email address or
+an SMS OTP to their phone number are common account recovery methods.
+* **Payment confirmation** In payment systems, some banks or credit card
+issuers request additional authentication from the payer for security reasons.
+SMS OTP is commonly used for that purpose.
+
+This post explains best practices to build an SMS OTP form for the above use
+cases.
+
+{% Aside 'caution' %}
+While this post discusses SMS OTP form best practices, be aware that SMS OTP is
+not the most secure method of authentication by itself because phone
+numbers can be recycled and sometimes hijacked. And [the concept of OTP
+itself is not phishing resistant](https://youtu.be/kGGMgEfSzMw?t=1133).
+
+If you are looking for better security, consider using
+[WebAuthn](https://www.w3.org/TR/webauthn-2/). Learn more about it from the talk
+"[What's new in sign-up & sign-in](https://goo.gle/webauthn-video)" at the
+Chrome Dev Summit 2019. {% endAside %}
+
+## Checklist
+
+To provide the best user experience with the SMS OTP, follow these steps:
+
+* Use the `<input>` element with:
+    * `type="text"`
+    * `inputmode="numeric"`
+    * `autocomplete="one-time-code"`
+* Use `@BOUND_DOMAIN #OTP_CODE` as the last line of the OTP SMS message.
+* Use the [Web OTP API](/web-otp/).
+
+## Use the `<input>` element
+
+Using a form with an `<input>` element is the most important best practice you
+can follow because it works in all browsers. Even if other suggestions from
+this post don't work in some browser, the user will still be able to enter and submit the OTP 
+manually.
+
+```html
+<form action="/verify-otp" method="POST">
+  <input type="text"
+         inputmode="numeric"
+         autocomplete="one-time-code"
+         pattern="\d{6}"
+         required>
+</form>
+```
+
+The following are a few ideas to ensure an input field gets the best out of
+browser functionality.
+
+{% Aside %}
+For more general form best practices, [Sam Dutton](/authors/samdutton/)'s
+[Sign-in form best practices](/sign-in-form-best-practices/) is a great starting
+point.
+{% endAside %}
+
+### `type="text"`
+
+Because OTPs are usually five or six digit numbers, using
+`type="number"` for an input field might seem intuitive because it changes the mobile 
+keyboard to numbers only. This is not recommended because the browser expects an
+input field to be a countable number rather than a sequence of multiple numbers,
+which can cause unexpected behavior. Using `type="number"` causes up and down
+buttons to be displayed beside the input field; pressing these buttons
+increments or decrements the number and may remove preceding zeros.
```

Comment:
```suggestion
Since OTPs are usually five or six digit numbers, using
`type="number"` for an input field might seem intuitive because it changes the mobile 
keyboard to numbers only. This is not recommended because the browser expects an
input field to be a countable number rather than a sequence of multiple numbers,
which can cause unexpected behavior. Using `type="number"` causes up and down
buttons to be displayed beside the input field; pressing these buttons
increments or decrements the number and may remove preceding zeros.
```